pcdsdevices/lasers/qmini.py

```diff
@@ -35,12 +38,14 @@ class QminiSpectrometer(Device):
                     kind='config')
     trig_enable = Cpt(EpicsSignal, ':GET_TRIG_ENABLE',
                       write_pv=':SET_TRIG_ENABLE', kind='config')
-    scan_rate = Cpt(EpicsSignal, ':START_EXPOSURE.SCAN', kind='config')
+    scan_rate = Cpt(EpicsSignal, ':GET_SPECTRUM.SCAN', kind='config')
     reset = Cpt(EpicsSignal, ':CLEAR_SPECTROMETER', kind='config')
+    set_metadata(reset, dict(variety='command-proc', value=1))
     spectrum = Cpt(EpicsSignalRO, ':SPECTRUM', kind='normal')
     wavelengths = Cpt(EpicsSignalRO, ':WAVELENGTHS', kind='normal')
 
     model = Cpt(EpicsSignalRO, ':MODEL_CODE', kind='config')
+    set_metadata(model, dict(variety='scalar', display_format='hex'))
```

Comment:
This didn't appear to work, but not a big deal.

Reply:
Hmm, if you have a chance, give this branch a try: https://github.com/pcdshub/typhos/pull/431

Reply:
I tried that, with no luck. For reference, when converting to hex in EDM, I get the expected model code 276E0208:

![image](https://user-images.githubusercontent.com/29102919/112234832-98687700-8bfa-11eb-92f1-2b9fc3c15b6b.png)

Reply:
Hmm, OK - thanks for giving it a shot. Will have to look into it more later.